specification/cognitiveservices/data-plane/Form/readme.md

```diff
@@ -0,0 +1,72 @@
+# Cognitive Services Form SDK
```

Comment:
`Form Recognizer` or `FormRecognizer` should be used in everywhere instead of `Form`
Include file name, folder name, namespace, etc....

Reply:
Updated to use FormRecognizer.